tox_virtualenv_no_download.py

```diff
@@ -1,5 +1,4 @@
 from __future__ import absolute_import
-from __future__ import unicode_literals
```

Comment:
I'd rather see explicit native literals instead, you can make a native literal via `str('string literal')`

Reply:
Ugh... fine.  Just release it :)